src/ert/cli/main.py

```diff
@@ -97,7 +97,37 @@ def run_cli(args: Namespace, plugin_manager: ErtPluginManager | None = None) ->
         )
     except ValueError as e:
         raise ErtCliError(f"{args.mode} was not valid, failed with: {e}") from e
-
+    if (
```

Comment:
as agreed, let's put this code into a separate function and then we will have an easier way to test it.